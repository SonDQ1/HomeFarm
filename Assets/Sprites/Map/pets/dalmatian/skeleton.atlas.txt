
skeleton.png
format: RGBA8888
filter: Linear,Linear
repeat: none
body
  rotate: false
  xy: 2, 14
  size: 149, 112
  orig: 149, 112
  offset: 0, 0
  index: -1
ear
  rotate: false
  xy: 430, 91
  size: 25, 35
  orig: 25, 35
  offset: 0, 0
  index: -1
ear2
  rotate: false
  xy: 399, 88
  size: 29, 38
  orig: 29, 38
  offset: 0, 0
  index: -1
head
  rotate: false
  xy: 153, 36
  size: 82, 90
  orig: 82, 90
  offset: 0, 0
  index: -1
leg_back
  rotate: false
  xy: 296, 48
  size: 36, 78
  orig: 36, 78
  offset: 0, 0
  index: -1
leg_back2
  rotate: false
  xy: 296, 48
  size: 36, 78
  orig: 36, 78
  offset: 0, 0
  index: -1
leg_front
  rotate: true
  xy: 334, 55
  size: 26, 63
  orig: 26, 63
  offset: 0, 0
  index: -1
leg_front2
  rotate: true
  xy: 334, 55
  size: 26, 63
  orig: 26, 63
  offset: 0, 0
  index: -1
tail
  rotate: false
  xy: 334, 83
  size: 63, 43
  orig: 63, 43
  offset: 0, 0
  index: -1
thigh_back
  rotate: false
  xy: 237, 46
  size: 57, 80
  orig: 57, 80
  offset: 0, 0
  index: -1
thigh_back2
  rotate: false
  xy: 237, 46
  size: 57, 80
  orig: 57, 80
  offset: 0, 0
  index: -1
thigh_front
  rotate: true
  xy: 153, 2
  size: 32, 66
  orig: 32, 66
  offset: 0, 0
  index: -1
thigh_front2
  rotate: true
  xy: 153, 2
  size: 32, 66
  orig: 32, 66
  offset: 0, 0
  index: -1
