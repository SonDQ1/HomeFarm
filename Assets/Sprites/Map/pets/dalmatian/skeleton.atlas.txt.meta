fileFormatVersion: 2
guid: 7a0918adfabf59f46a432c5de7697f18
timeCreated: 1550284193
licenseType: Pro
TextScriptImporter:
  userData: 
  assetBundleName: 
  assetBundleVariant: 
