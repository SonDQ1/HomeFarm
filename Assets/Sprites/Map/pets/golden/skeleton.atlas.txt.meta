fileFormatVersion: 2
guid: 6fed08164c9b6ba438574fba4238e01e
timeCreated: 1550284378
licenseType: Pro
TextScriptImporter:
  userData: 
  assetBundleName: 
  assetBundleVariant: 
