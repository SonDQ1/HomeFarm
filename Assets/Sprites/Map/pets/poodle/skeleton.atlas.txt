
skeleton.png
format: RGBA8888
filter: Linear,Linear
repeat: none
body1
  rotate: false
  xy: 2, 30
  size: 90, 87
  orig: 90, 87
  offset: 0, 0
  index: -1
body2
  rotate: false
  xy: 94, 55
  size: 111, 62
  orig: 111, 62
  offset: 0, 0
  index: -1
ear
  rotate: true
  xy: 94, 14
  size: 39, 54
  orig: 39, 54
  offset: 0, 0
  index: -1
ear2
  rotate: true
  xy: 94, 14
  size: 39, 54
  orig: 39, 54
  offset: 0, 0
  index: -1
hat
  rotate: false
  xy: 207, 63
  size: 66, 54
  orig: 66, 54
  offset: 0, 0
  index: -1
head
  rotate: false
  xy: 335, 69
  size: 55, 48
  orig: 55, 48
  offset: 0, 0
  index: -1
leg_back
  rotate: true
  xy: 392, 91
  size: 26, 73
  orig: 26, 73
  offset: 0, 0
  index: -1
leg_back2
  rotate: true
  xy: 392, 91
  size: 26, 73
  orig: 26, 73
  offset: 0, 0
  index: -1
leg_front
  rotate: true
  xy: 2, 2
  size: 26, 77
  orig: 26, 77
  offset: 0, 0
  index: -1
leg_front2
  rotate: true
  xy: 2, 2
  size: 26, 77
  orig: 26, 77
  offset: 0, 0
  index: -1
tail
  rotate: true
  xy: 275, 64
  size: 53, 58
  orig: 53, 58
  offset: 0, 0
  index: -1
thigh_back
  rotate: false
  xy: 150, 17
  size: 29, 36
  orig: 29, 36
  offset: 0, 0
  index: -1
thigh_back2
  rotate: false
  xy: 150, 17
  size: 29, 36
  orig: 29, 36
  offset: 0, 0
  index: -1
thigh_front
  rotate: true
  xy: 467, 94
  size: 23, 39
  orig: 23, 39
  offset: 0, 0
  index: -1
thigh_front2
  rotate: true
  xy: 467, 94
  size: 23, 39
  orig: 23, 39
  offset: 0, 0
  index: -1
