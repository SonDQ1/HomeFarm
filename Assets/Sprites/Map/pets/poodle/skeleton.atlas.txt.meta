fileFormatVersion: 2
guid: 0e5cadaf5e0e1744fa5e156136ce3ded
timeCreated: 1550284454
licenseType: Pro
TextScriptImporter:
  userData: 
  assetBundleName: 
  assetBundleVariant: 
