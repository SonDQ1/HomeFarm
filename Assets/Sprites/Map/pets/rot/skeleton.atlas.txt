
skeleton.png
format: RGBA8888
filter: Linear,Linear
repeat: none
body
  rotate: false
  xy: 2, 115
  size: 187, 139
  orig: 187, 139
  offset: 0, 0
  index: -1
ear
  rotate: true
  xy: 122, 5
  size: 37, 41
  orig: 37, 41
  offset: 0, 0
  index: -1
ear2
  rotate: false
  xy: 84, 2
  size: 36, 40
  orig: 36, 40
  offset: 0, 0
  index: -1
head
  rotate: false
  xy: 2, 34
  size: 80, 79
  orig: 80, 79
  offset: 0, 0
  index: -1
leg_back
  rotate: true
  xy: 170, 52
  size: 36, 73
  orig: 36, 73
  offset: 0, 0
  index: -1
leg_back2
  rotate: true
  xy: 170, 52
  size: 36, 73
  orig: 36, 73
  offset: 0, 0
  index: -1
leg_front
  rotate: true
  xy: 2, 2
  size: 30, 63
  orig: 30, 63
  offset: 0, 0
  index: -1
leg_front2
  rotate: true
  xy: 2, 2
  size: 30, 63
  orig: 30, 63
  offset: 0, 0
  index: -1
tail
  rotate: true
  xy: 191, 90
  size: 76, 47
  orig: 76, 47
  offset: 0, 0
  index: -1
thigh_back
  rotate: true
  xy: 84, 44
  size: 69, 84
  orig: 69, 84
  offset: 0, 0
  index: -1
thigh_back2
  rotate: true
  xy: 84, 44
  size: 69, 84
  orig: 69, 84
  offset: 0, 0
  index: -1
thigh_front
  rotate: false
  xy: 191, 168
  size: 61, 86
  orig: 61, 86
  offset: 0, 0
  index: -1
thigh_front2
  rotate: false
  xy: 191, 168
  size: 61, 86
  orig: 61, 86
  offset: 0, 0
  index: -1
