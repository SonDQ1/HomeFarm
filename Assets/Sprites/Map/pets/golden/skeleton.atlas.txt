
skeleton.png
format: RGBA8888
filter: Linear,Linear
repeat: none
body
  rotate: false
  xy: 2, 91
  size: 174, 140
  orig: 174, 140
  offset: 0, 0
  index: -1
ear
  rotate: true
  xy: 64, 2
  size: 34, 56
  orig: 34, 56
  offset: 0, 0
  index: -1
ear2
  rotate: false
  xy: 413, 175
  size: 36, 56
  orig: 36, 56
  offset: 0, 0
  index: -1
head
  rotate: true
  xy: 178, 120
  size: 111, 147
  orig: 111, 147
  offset: 0, 0
  index: -1
leg_back
  rotate: true
  xy: 451, 199
  size: 32, 59
  orig: 32, 59
  offset: 0, 0
  index: -1
leg_back2
  rotate: true
  xy: 451, 199
  size: 32, 59
  orig: 32, 59
  offset: 0, 0
  index: -1
leg_front
  rotate: false
  xy: 381, 142
  size: 30, 89
  orig: 30, 89
  offset: 0, 0
  index: -1
leg_front2
  rotate: false
  xy: 381, 142
  size: 30, 89
  orig: 30, 89
  offset: 0, 0
  index: -1
tail
  rotate: false
  xy: 64, 38
  size: 76, 51
  orig: 76, 51
  offset: 0, 0
  index: -1
thigh_back
  rotate: false
  xy: 2, 7
  size: 60, 82
  orig: 60, 82
  offset: 0, 0
  index: -1
thigh_back2
  rotate: false
  xy: 2, 7
  size: 60, 82
  orig: 60, 82
  offset: 0, 0
  index: -1
thigh_front
  rotate: false
  xy: 327, 142
  size: 52, 89
  orig: 52, 89
  offset: 0, 0
  index: -1
thigh_front2
  rotate: false
  xy: 327, 142
  size: 52, 89
  orig: 52, 89
  offset: 0, 0
  index: -1
